src/DGtal/io/doc/moduleBoard2D.dox

```diff
@@ -1,5 +1,5 @@
 
```

Comment:
in the image DGtal/io/doc/images/colormaps.png a part of the text on the left looks hidden, if you have the possibility to shift it in the left it should be nice:
![colormaps](https://user-images.githubusercontent.com/772865/34083000-15c53e88-e369-11e7-8c11-b0ecb437a755.png)

Reply:
Thanks.. fixed